content/en/docs/Writing policies/jmespath.md

```diff
@@ -408,6 +408,100 @@ spec:
 </p>
 </details>
 
+### Sum
+
+<details><summary>Expand</summary>
+
+<p>
+
+The `sum()` filter very simply adds a list of elements and returns a result. The exception to this rule is `sum()`, which is described here. The official JMESPath library does not have the most fundamental arithmetic operators, such as add, subtract, multiply, and divide. `Sum()`, which is defined in our own JMESPath filters, is beneficial as a streamlined filter when a list of any type of values is needed to be summed, but [`sum()`](https://jmespath.org/specification.html#sum) in the JMESPath library is handy in that it accepts an array of integers as an input. See how sum() differs from the add() filter, which is used to add the values of two fields, in this case. If you want to sum the values in a list of fields, use sum() instead.
+
+`sum()` is value-aware (based on the formatting used for the inputs) just as `add()` and is capable of adding list of numbers, quantities, and durations without any form of unit conversion.
+
+Arithmetic filters like `sum()` currently accept inputs in the following formats.
+
+* Number (ex., \`10\`)
+* Quantity (ex., '10Mi')
+* Duration (ex., '10h')
+
+Note that how the inputs are enclosed determines how Kyverno interprets their type. Numbers enclosed in back ticks are scalar values while quantities and durations are enclosed in single quotes thus treating them as strings. Using the correct enclosing character is important because, in Kubernetes "regular" numbers are treated implicitly as units of measure. The number written \`10\` is interpreted as an integer or "the number ten" whereas '10' is interpreted as a string or "ten bytes". See the [Formatting](#formatting) section above for more details.
+
+| Input                 | Output   |
+|-----------------------|----------|
+| [Number]              | Number   |
+| [Quantity or Number]  | Quantity |
+| [Duration or Number]  | Duration |
```

Comment:
Please use the established formatting, for ex: https://github.com/kyverno/website/blob/bd55c2e78cfb4bcfd8d0903a4b079af69904897d/content/en/docs/Writing%20policies/jmespath.md?plain=1#L363-L367

Reply:
But this format is defined for the function with two inputs.

Reply:
The heading would be "Input 1" and rather than square brackets around the name of the input, just say "Array/Number" etc.

Reply:
resolved

Reply:
resolved